core/invoker/src/main/resources/application.conf

```diff
@@ -28,6 +28,7 @@ whisk {
       inspect: 1 minute
       pause: 10 seconds
       unpause: 10 seconds
+      version: 1 minute
```

Comment:
While I think it's a great idea to add a timeout for the version command, I also think that a minute is far too long. It's only run once during invoker startup and should never be run again because the version cannot change while the invoker is running.

Even `pause` and `unpause` are limited to 10 seconds which can be tight on a loaded invoker.

I suggest to use at most 10 seconds here. Even in a slow Travis env, this should never take longer.

Reply:
done